### PR TITLE
Bug 1502075 - stop using v1 treeherder routes in integration tests

### PR DESCRIPTION
**Pull request by petemoore**
Note, the commit hash is invented.

See [bug 1502075](https://bugzil.la/1502075) for context.

### CODE REVIEW

File: src/test/java/org/mozilla/taskcluster/client/APITest.java [hunk 1]
```diff
@@ -128,8 +128,7 @@ public void testDefineTask() {
         td.payload = payload;
         td.provisionerId = "win-provisioner";
         td.retries = 5;
-        td.routes = new String[] { "tc-treeherder.mozilla-inbound.bcf29c305519d6e120b2e4d3b8aa33baaf5f0163",
-                "tc-treeherder-stage.mozilla-inbound.bcf29c305519d6e120b2e4d3b8aa33baaf5f0163" };
+        td.routes = new String[] { "tc-treeherder.v2.mozilla-inbound.27cb49253533443d31187bcd40360669f5110e1a.109189" };
```
[the remark this answers is not in the document]
Comment (petemoore) [reply src/test/java/org/mozilla/taskcluster/client/APITest.java]: See above:

> Note, the commit hash is invented.

;-)

[the remark this answers is not in the document]
Comment (djmitche) [reply src/test/java/org/mozilla/taskcluster/client/APITest.java]: Why have the route at all, then?

[the remark this answers is not in the document]
Comment (petemoore) [reply src/test/java/org/mozilla/taskcluster/client/APITest.java]: To make sure that the client is able to generate tasks that contain a route.

[the remark this answers is not in the document]
Comment (djmitche) [reply src/test/java/org/mozilla/taskcluster/client/APITest.java]: Perhaps a better choice in that case would be a garbage route, that nothing is listening to.  Maybe just `garbage.tc-client-java.test`?  That way the test data isn't getting passed through two other production services (tc-treeherder, and treeherder itself) that may potentially get confused by it (as happened here).

**Pull request merged**